libraries/AP_Motors/AP_MotorsHeli.cpp

```diff
@@ -193,6 +193,38 @@ const AP_Param::GroupInfo AP_MotorsHeli::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("RSC_THRCRV_100", 24, AP_MotorsHeli, _rsc_thrcrv[4], AP_MOTORS_HELI_RSC_THRCRV_100_DEFAULT),
 
+    // @Param: RSC_GOV_SET
```

Comment:
It might be a good idea to move the RSC_ parameters off into a separate group so they appear/disappear if the feature is enabled/disabled.  This is an existing thing though.

Reply:
this is done in a follow-up commit so marking as "resolved".